routing/transit_graph.cpp

```diff
@@ -188,15 +190,17 @@ transit::Edge const & TransitGraph::GetEdge(Segment const & segment) const
   return it->second;
 }
 
-Segment TransitGraph::GetTransitSegment(uint32_t segmentIdx) const
+Segment TransitGraph::GetTransitSegment(uint32_t featureId) const
 {
-  return Segment(m_mwmId, kTransitFeatureId, segmentIdx, false);
+  CHECK(IsTransitFeature(featureId),("Feature id is out of transit id interval."));
```

Comment:
space after |,|?
I guess it's worth printing |featureId| in this check.

Reply:
done